step-definitions/utils/auth.js

```diff
@@ -85,7 +85,7 @@ const logInAs = async function (username) {
     await logInNormallyAs(this, username);
   }
 
-  expect(await this.browser.pageText()).toContain(username);
+  expect(await this.browser.pageText()).toMatch(new RegExp(username, "i"));
```

Comment:
Not sure how this is only popping up now, but we had an issue around casing with usernames - this matches insensitively.